plugins/woocommerce-admin/client/two-column-tasks/task-list-completion-slot/utils.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { Slot, Fill } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { createOrderedChildren, sortFillsByOrder } from '../../utils';
+
+type WooTaskListCompletionItemProps = {
+	children: React.ReactNode;
+	order?: number;
+};
+
+export const EXPERIMENTAL_WC_TASK_LIST_COMPLETION_SLOT_NAME =
+	'woocommerce_task_list-completion_item';
+/**
+ * Create a Fill for extensions to add items to the WooCommerce Admin Task List completion component slot.
+ *
+ * @slotFill WooTaskListCompletionItem
+ * @scope woocommerce-admin
+ * @example
+ * const MyTasklistCompletionItem = () => (
+ * <WooTaskListCompletionItem>My Task List completion item</WooTaskListCompletionItem>
+ * );
+ *
+ * registerPlugin( 'my-extension', {
+ * render: MyTasklistCompletionItem,
+ * scope: 'woocommerce-admin',
+ * } );
+ * @param {Object} param0
+ * @param {Array}  param0.children - Node children.
+ * @param {Array}  param0.order    - Node order.
+ */
+export const WooTaskListCompletionItem = ( {
```

Comment:
@rjchow I've been thinking, `WooTaskListCompletionItem` sounds a little bit misrepresenting since it's not an "Item" per-se. Does `WooTaskListCompletion` sound ok to you?

Reply:
Sure thing, not sure why I named it that way!

Typescript :chefskiss
Just a single F2->change string and it's fixed everywhere!